src/foam/nanos/Threadpool.js

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
/**
 * @license
 * Copyright 2020 The FOAM Authors. All Rights Reserved.
 * http://www.apache.org/licenses/LICENSE-2.0
 */